crates/katana/executor/src/implementation/blockifier/mod.rs

```diff
@@ -158,13 +160,13 @@ impl<'a> BlockExecutor<'a> for StarknetVMProcessor<'a> {
                     crate::utils::log_events(receipt.events());
 
                     if let Some(reason) = receipt.revert_reason() {
-                        info!(target: "executor", "transaction reverted: {reason}");
+                        info!(target: LOG_TARGET, reason = %reason, "Transaction reverted.");
                     }
 
                     ExecutionResult::new_success(receipt, trace, fee)
                 }
                 Err(e) => {
-                    info!(target: "executor", "transaction execution failed: {e}");
+                    error!(target: LOG_TARGET, error = %e, "Executing transaction.");
```

Comment:
not sure if this should be `error` as i would assume `error` is for when there's an error in the actual task. failing transaction doesn't indicate that the execution task itself failed